latest failing in ec2

```
[    7.182589] ignition[2018]: INFO     : GET error: Get http://169.254.169.254/2009-04-04/user-data: dial tcp 169.254.169.254:80: connect: network is unreachable
```

until we time out.  The qcow2 boots fine for me with `coreos-assembler run -d /path/to/rhcos-4.0.5743-qemu.qcow2`, and I have networking post-boot.

Maybe something related to new Ignition?

Not sure if the same cause, but we don't make it past the basic tests:
```
--- FAIL: rhcos.basic (52.72s)
        harness.go:434: Cluster failed starting machines: machine "c3d40f67-42b7-492f-9e71 4464d8f502e9" failed basic checks: some systemd units failed:

crio.service loaded failed failed Open Container Initiative Daemon
```

--- a/cloud.ks
+++ b/cloud.ks
@@ -25,7 +25,7 @@ clearpart --initlabel --all
 #  - ip=dhcp           # how to get network
 #  - rd.neednet=1      # tell dracut we need network
 #  - $coreos_firstboot # This is actually a GRUB variable
-bootloader --timeout=1 --append="no_timer_check console=ttyS0,115200n8 console=tty0 net.ifnames=0 biosdevname=0 ip=dhcp rd.neednet=1 rw $coreos_firstboot"
+bootloader --timeout=1 --append="no_timer_check console=ttyS0,115200n8 console=tty0 net.ifnames=0 biosdevname=0 ip=eth0:dhcp rd.neednet=1 rw $coreos_firstboot"
 
 part /boot --size=300 --fstype="xfs" --label=boot
 part / --size=3000 --fstype="xfs" --label=root --grow

